Source/WebCore/rendering/BreakLines.h

```diff
@@ -190,36 +194,39 @@ inline size_t BreakLines::nextBreakablePosition(CachedLineBreakIteratorFactory&
         }
 
         // Non-ASCII rapid lookup.
-        if (words != WordBreakBehavior::AutoPhrase) {
+        if constexpr (words != WordBreakBehavior::AutoPhrase) {
             if (!before.type)
                 before.type = classify<shortcutRules, nonBreakingSpaceBehavior>(before);
             after.type = classify<shortcutRules, nonBreakingSpaceBehavior>(after);
             // Short-circuit the commonest cases: letter + letter.
             int pair = before.type | after.type;
-            if (pair == (kAL | kAL)) {
-                if (words == WordBreakBehavior::BreakAll)
+            // SP+AL SP+QU AL+AL AL+QU QU+QU QU+AL (after's SP is already filtered out).
```

Comment:
Nit: List the AL + AL combination first, since it's the most important one here. :)

Reply:
Done.

Reply:
I meant in the comment! ^_^;;

Reply:
Ordering of conditionals in this particular function is not nitpicking, it's perf hacking. ;)